test/ccapi/unittest_ccapi.cpp

```diff
@@ -152,36 +153,38 @@ TEST(ccapi_dataset, construct_02_p) {
   EXPECT_NO_THROW(ml::train::createDataset(ml::train::DatasetType::FILE));
 }
 
```

Comment:
how about `using nntrainer::IniSection?`

Reply:
Not a big deal though, I wonder if i should invest more time to change. I think qualified name works just fine as well.

Reply:
Either is fine.